Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -84,5 +84,5 @@
    scalersInputTag = cms.InputTag( "rawDataRepacker" )
 )
 
-DigiToApproxClusterRawTask = cms.Task(siStripDigisHLT,siStripZeroSuppressionHLT,hltScalersRawToDigi,hltBeamSpotProducer,siStripClustersHLT,hltSiStripClusters2ApproxClusters,rawPrimeDataRepacker)
+DigiToApproxClusterRawTask = cms.Task(hltSiStripRawToDigi,siStripZeroSuppressionHLT,hltScalersRawToDigi,hltBeamSpotProducer,siStripClustersHLT,hltSiStripClusters2ApproxClusters,rawPrimeDataRepacker)
```

Comment:
as a side note, we could actually rename *all* the HLT mock-up collections:
   * `siStripZeroSuppressionHLT` -> `hltSiStripZeroSuppression`
   * `siStripClustersHLT` -> `hltSiStripClusterizerForRawPrime`  
   
even though since the event products are not persisted, it's not strictly mandatory to be in synch with the content of the HLT menu. I let the reviewers decide.